recipes/eko/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "eko" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 02fba5228d3830576c70e4bf766d5014096ca342fa4ef7c7317978cc75558002
+
+build:
+  entry_points:
+    - ekonav = ekomark.navigator:launch_navigator
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - pip
+  run:
```

Comment:
This is not a blocker but it might also make sense to add run_constrained here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained

Basically, it doesn't install the specified packages but if they are installed by the user spearately, they can only be installed in the range that you specify. 
This might make sense for the optional packages here: https://github.com/N3PDF/eko/blob/341e6323e680d2320ae4400da61c92a445160416/pyproject.toml#L43-L45

Reply:
Since it depends on `banana` they are installed anway, but I can add them explicitly as well if that's the right way to do it.

Reply:
Ahh ok no then it's fine. It's not a necessity anyways.